src/Refactoring-Core/RBIsNotGlobal.class.st

```diff
@@ -0,0 +1,32 @@
+"
+I check if no global in the `model` is named the same as `name`
+"
+Class {
+	#name : 'RBIsNotGlobal',
```

Comment:
I have already implemented similar class in a PR that is already integrated. It's called `RBNameIsGlobalCondition` and has this implementation. So we can avoid creating this and just do `(RBNameIsGlobalCondition new model: model className: name) not`.

Reply:
done in new commit :)